projects/core/tsconfig.lib.prod.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "./tsconfig.lib.json",
+  "compilerOptions": {
+    "declarationMap": false
+  },
   "angularCompilerOptions": {
     "enableIvy": false
```

Comment:
Should this be removed?

Reply:
I think it makes sense to disable source map generation for the production builds. I'm good with leaving this as-is.